orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/TrafficGuard.java

```diff
@@ -280,6 +280,22 @@ public void verifyTrafficRemoval(
             .reduce(0, Integer::sum);
 
     int futureCapacity = currentCapacity - capacityGoingAway;
+
+    int someDesiredSize = someServerGroup.getCapacity().getDesired();
+    if (futureCapacity > 0
+        && serverGroupsGoingAway.size() > 1
+        && serverGroupsGoingAway.stream().allMatch(sg -> sg.getCapacity().isPinned())
+        && serverGroupsGoingAway.stream()
+            .allMatch(sg -> sg.getCapacity().getDesired() == someDesiredSize)) {
+      log.debug(
```

Comment:
def `info` level to me (at least)

Reply:
also, maybe `...with all server groups pinned with matching capacity`? would be quicker to tell whats up in logs if this ever bites us

Reply:
made this `debug` for consistency with all the other logs from the `TrafficGuard` class. Would you motion for all of them to become `info` or just this one?

Reply:
hm... i see. leave as `debug` then